### ISSUE TITLE
FMOD failing to initialize (Dying Light)

### ISSUE THREAD

**Issue by rowbawts** [flathub/com.valvesoftware.Steam#36]
Game works perfectly other than there is no sound at all, and in the games log I see "Error! FMOD System failed to initialize." I think this may be an issue with the Flatpak specifically if not close this.

```
{21:36:13.820} INFO: [INFO] > >>>>> Audio init
{21:36:13.903} INFO: [INFO] > FMOD::Driver: default
{21:36:13.951} ERRR: [ERRR] > Error! FMOD System failed to initialize.
```

**Comment by TingPing** [comment]
Did you find a solution?

**Comment by rowbawts** [comment]
Yes but it was my fault I was using an older version of Flatpak once I updated to 0.10 it worked, but in between upgrading that and the sound working I made other changes so I'm not sure which fixed it.

**Comment by rowbawts** [comment]
@TingPing It's actually not really resolved not sure if it's an issue with the Flatpak or the game but the sound is now just a bunch of static.

**Comment by nanonyme** [comment]
Any update on this?

**Comment by nanonyme** [comment]
This game has a dependency on Mesa 17.2 as it uses OpenGL compatibility contexts. Blocks any testing on non-nVidia

**Comment by nanonyme** [comment]
Runtime now has Mesa 17.2. This needs to be tested again
